src/JSInterop/Microsoft.JSInterop/src/JSRuntime.cs

```diff
@@ -88,7 +88,7 @@ public async ValueTask<TValue> InvokeAsync<TValue>(string identifier, object[] a
         public ValueTask<TValue> InvokeAsync<TValue>(string identifier, CancellationToken cancellationToken, object[] args)
         {
             var taskId = Interlocked.Increment(ref _nextPendingTaskId);
-            var tcs = new TaskCompletionSource<TValue>(TaskContinuationOptions.RunContinuationsAsynchronously);
+            var tcs = new TaskCompletionSource<TValue>(TaskCreationOptions.RunContinuationsAsynchronously);
```

Comment:
Looks like a bunch of tests relied on this being inline instead of async

Reply:
ping @pranavkm @captainsafia can we get this resolved today?